docs/products/access/release-notes/index.md

```diff
@@ -8,6 +8,22 @@ The releases are backward compatible. However, we will extend the API contract (
 
 In the release notes, we mention new features and bug fixes. If anything is unclear, feel free to contact [OneWelcome Support](https://support.onewelcome.com).
 
+## Release date 2024-01-22
+
+### Features
+* Added support for the [OAuth Device Flow](../api-reference/description-oauth-endpoint.md#oauth-device-flow-extension)
+
+### Improvements
+* We improved the security of our session cookies by adding a `__Host` prefix to them.
+
+  We included both the legacy and the new secure cookies for existing customers to keep the existing sessions.
+
+  The legacy cookies can be disabled in the Access Admin, by going to `Configuration` → `System` → `Cookies` and turning off the `Add fallback cookie` option.
+  The existing users sessions will not be impacted.
```

Comment:
Are you sure about this last part? I thought enabling this would need to be done in a quiet moment as it would have impact.

Reply:
That's why we have two cookies. Before this update, it was `SESSION` and `LEGACY_SESSION`. Now, after the update, we have `__Host-ACCESS_ENGINE_SESSION` and `LEGACY_SESSION.`

The session is valid for 60 minutes, so an hour after our release, all user sessions should be invalid or use new cookies.

Once the `Add fallback cookie` is disabled. We only read and write the `__Host-ACCESS_ENGINE_SESSION`.

Reply:
I run some tests on the playground to be sure.
So if the user prolongs its session with the `prompt=none` request, the new cookie isn't set. Only when the new session is created.
The default max session length is 7 days, so 7 days from today, all users will have new sessions (and new cookies), and it will be safe to switch the `Add fallback cookie` option.
But we also clear the cache every two nights or so.